### ISSUE TITLE
D&D description is not displayed correctly

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-android#4087]
Screenshot:
![Image](https://github.com/user-attachments/assets/b1620d71-66c1-43ea-b514-07bbf0b54caf)

**Comment by kelson42** [comment]
Probably a problem with MWoffliner

**Comment by MohitMaliFtechiz** [comment]
@kelson42 I don't know if it is an MWoffiner problem or not as the description has the HTML entry in it. But It is a Kiwix app problem the description is "D\&amp;D Wiki" which has the HTML entry. But Kiwix is not properly decoding the HTML entries before setting it to the textView. I have fixed this issue in https://github.com/kiwix/kiwix-android/pull/4111.

**Comment by MohitMaliFtechiz** [comment]
@kelson42 No, we don't want to decode the png in the Kiwix. The problem is on MWoffliner as it is puting the HTML in the description, but at least Kiwix should decode the HTML for the title and description if any comes due to this type of bug to avoid this type of visual error.

**Comment by kelson42** [comment]
@MohitMaliFtechiz No, bug should be fixed at the right place. Fixing problems at the wrong place is the best way to create strange behaviours in edge cases and future bugs.

**Comment by kelson42** [comment]
Kamino closed and cloned this issue to [openzim/mwoffliner](https://github.com/openzim/mwoffliner/issues/2164)
